package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "android": "react-native run-android",
+    "pods": "cd ios && pod install",
```

Comment:
can you please include && cd .. here so that the person is back in the root folder?

Reply:
It does this automatically! No need to add cd ../

Reply:
Oh i didn't know that, that's handy!